x-pack/legacy/plugins/infra/public/pages/logs/analysis/page.tsx

```diff
@@ -32,6 +34,11 @@ export const AnalysisPage = () => {
     spaceId,
     timeField: source ? source.configuration.fields.timestamp : '',
   });
+  const { hasLogAnalysisCapabilites } = useLogAnalysisCapabilities();
+  if (!hasLogAnalysisCapabilites) {
```

Comment:
This violates the invariant of the number of rendered hooks.

Reply:
Yep, as above, this will potentially lead to `Error: Rendered more hooks than during the previous render.`.